nautobot/dcim/views.py

```diff
@@ -2139,6 +2145,38 @@ class DeviceBulkDeleteView(generic.BulkDeleteView):
     table = tables.DeviceTable
 
 
+class DeviceWirelessView(generic.ObjectView):
+    queryset = Device.objects.all()
+    template_name = "dcim/device/wireless.html"
+
+    def get_extra_context(self, request, instance):
+        access_point_group = instance.access_point_group
+        wireless_networks = AccessPointGroupWirelessNetworkAssignment.objects.filter(
+            access_point_group=access_point_group
+        ).select_related("wireless_network", "access_point_group", "vlan")
+        wireless_networks_table = AccessPointGroupWirelessNetworkAssignmentTable(
+            data=wireless_networks, user=request.user, orderable=False
+        )
+        wireless_networks_table.columns.hide("access_point_group")
+        wireless_networks_table.columns.hide("controller")
+        RequestConfig(request, paginate={"per_page": get_paginate_count(request)}).configure(wireless_networks_table)
```

Comment:
Not certain but maybe we should be including `paginator_class: EnhancedPaginator` in these RequestConfig calls? Seems to be another thing we're not very self-consistent about throughout the code base.

Reply:
Reading the EnhancedPaginator, it seems we will likely want that functionality, I will make the changes.